sunpy/net/jsoc/jsoc.py

```diff
@@ -742,7 +742,7 @@ def _make_recordset(self, series, start_time='', end_time='', wavelength='',
             else:
                 timestr = ''
 
-        if wavelength:
+        if wavelength != '':
```

Comment:
What was the issue here?

Reply:
If wavelength is a Quantity, `if wavelength:` raises a warning, and will eventually fail, as in the future `Quantity` can't be treated like a bool.